packages/side-nav/src/SideNavItem.tsx

```diff
@@ -180,34 +163,39 @@ export interface SideNavItemProps {
  @param props.as When provided, the component will be rendered as the component or html tag indicated
  *  by this prop. Other additional props will be spread on the anchor element.
  */
-
 const SideNavItem: ExtendableBox<
   SideNavItemProps & { ref?: React.Ref<any> },
   'button'
 > = React.forwardRef((props: SideNavItemProps, forwardRef) => {
   const {
-    active = false,
+    active: activeProp,
     disabled = false,
     ariaCurrentValue = AriaCurrentValue.Page,
     className,
     children,
+    onClick: onClickProp,
+    glyph,
+    ...rest
   } = props;
+  const { usingKeyboard: showFocus } = useUsingKeyboardContext();
+  const active = typeof activeProp === 'boolean' && activeProp;
```

Comment:
if we default to false then we don't have to check for anything here, and we can just use the passed in prop

Reply:
Nice call. We couldn't do this before when we supported currentPath.